plugins/periskop/README.md

```diff
@@ -0,0 +1,34 @@
+# periskop
+
+[Periskop](https://periskop.io/) is a pull-based, language agnostic exception aggregator for microservice environments.
+
+![periskop-logo](https://i.imgur.com/z8BLePO.png)
+
+### `PeriskopErrorsTable`
+
+The Periskop Backstage Plugin exposes an entity tab component named `PeriskopErrorsTable`. Each of the entries in the table will direct you to the error details in your deployed Periskop instance location.
+
+![periskop-errors-card](./docs/periskop-plugin-screenshot.png)
+
+Now your plugin should be visible as a tab at the top of the entity pages.
+However, it warns of a missing `periskop.io/name` annotation.
+
+Add the annotation to your component descriptor file as shown in the highlighted example below:
+
+```yaml
+annotations:
+  periskop.io/name: '<THE NAME OF THE PERISKOP APP>'
+```
+
+### Locations
+
+The periskop plugin can be configured to fetch aggregated errors from multiple deployment locations. This is especially useful if you have a multi-zone deployment, or a federated setup and would like to drill deeper into a single instance of the federation. Each of the configured locations will be included in the plugin's UI via a dropdown on the errors table.
+
+The plugin requires to configure _at least one_ Periskop API location in the [app-config.yaml](https://github.com/backstage/backstage/blob/master/app-config.yaml):
+
+```yaml
+periskop:
+  locations:
+    - name: <name of the location>
+      host: <HTTP/S host for the Periskop API location>
```

Comment:
minor note: we have a couple of places in the config where we refer to things as `host` and in those places it strictly has to be a hostname _without_ https://, so to avoid confusion maybe you would like to name this "url" or so? up to you

Reply:
Changed